### PR TITLE
Ghost follow is now an orbit!

### PR DESCRIPTION
**Pull request by MrsTonedOne**

### CODE REVIEW

**Comment by Steelpoint** [comment]
Let me see people easier, what's not to like :+1:

**Comment by WJohn** [comment]
This is adorable.

**Comment by daman997** [comment]
do it :+1:

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1423,6 +1423,9 @@ B --><-- A
 /atom/movable/proc/orbit(atom/A, radius = 10, clockwise = 1, angle_increment = 15)
 	if(!istype(A))
 		return
+	if (orbiting)
+		stop_orbit()
+		sleep 1 //sadly this is the only way to ensure the original orbit proc stops and resets the mob's transform.
```
Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: spooky, didn't know it accepted sleep as a keyword instead of a proc.

**Comment by Anonus** [comment]
Bonus; this means following someone won't have a delay as the trail of ghosts tries to catch up.
:+1:

**Comment by Dorsisdwarf** [comment]
Can we haz gifs?

File: code/__HELPERS/unsorted.dm [hunk 2]
```diff
@@ -1423,6 +1423,11 @@ B --><-- A
 /atom/movable/proc/orbit(atom/A, radius = 10, clockwise = 1, angle_increment = 15)
 	if(!istype(A))
 		return
+	if (orbiting)
+		stop_orbit()
+		sleep(1) //sadly this is the only way to ensure the original orbit proc stops and resets the atom's transform.
+		if (orbiting || !istype(A)) //post sleep re-check
+			return
```
Comment (MrPerson) [inline code/__HELPERS/unsorted.dm]: You may want to make sure A hasn't been deleted instead of just doing an istype()

Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: ```istype()``` will fail on null, so if it's truly deleted it's fine.
```istype()``` obviously doesn't cover qdel-limbo, but the object still exists during that so it should probably be fine.

Comment (MrPerson) [inline code/__HELPERS/unsorted.dm]: I'll admit that being qdeleted within a decisecond is unlikely, but it could happen

Comment (Anonus) [inline code/__HELPERS/unsorted.dm]: If it can, it WILL.

**Comment by Iamgoofball** [comment]
rip omnighost
:+1:

File: code/__HELPERS/unsorted.dm (hunk 2 above)
Comment (MrStonedOne) [inline code/__HELPERS/unsorted.dm]: I fixed it in a much more practical way. (look down about 5 lines)

**Pull request merged**